Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -266,10 +275,26 @@ class SignalAttachment: NSObject {
         return SignalAttachment.audioUTISet.contains(dataUTI)
     }
 
+    public var isText: Bool {
+        return SignalAttachment.textUTISet.contains(dataUTI)
+    }
+
     public class func pasteboardHasPossibleAttachment() -> Bool {
         return UIPasteboard.general.numberOfItems > 0
     }
 
+    public class func pasteboardHasText() -> Bool {
+        if UIPasteboard.general.numberOfItems < 1 {
+            return false
+        }
+        let itemSet = IndexSet(integer:0)
+        guard let pasteboardUTITypes = UIPasteboard.general.types(forItemSet:itemSet) else {
+            return false
+        }
+        let pasteboardUTISet = Set<String>(pasteboardUTITypes[0])
+        return pasteboardUTISet.intersection(textUTISet).count > 0
```

Comment:
Oh, yeah I misread your comment.  But I didn't use that method because I didn't think it existed. And trying to use it, I get a build error.

![screen shot 2017-04-26 at 4 05 28 pm](https://cloud.githubusercontent.com/assets/625803/25454356/3245be9e-2a9a-11e7-9723-6ea8c5091fce.png)